tests/unit/metadata/test_multi_table.py

```diff
@@ -0,0 +1,237 @@
+"""Test Multi Table Metadata."""
+
+from unittest.mock import Mock, patch
+
+from sdv.metadata.multi_table import MultiTableMetadata
+
+
+class TestMultiTableMetadata:
+    """Test ``MultiTableMetadata`` class."""
+
+    def test___init__(self):
+        """Test the ``__init__`` method of ``MultiTableMetadata``."""
+        # Run
+        instance = MultiTableMetadata()
+
+        # Assert
+        assert instance._tables == {}
+        assert instance._relationships == []
+
+    def test_to_dict(self):
+        """Test the ``to_dict`` method of ``MultiTableMetadata``.
+
+        Setup:
+            - Instance of ``MultiTableMetadata``.
+            - Add mocked values to ``instance._tables`` and ``instance._relationships``.
+        Mock:
+            - Mock ``SingleTableMetadata`` like object to ``instance._tables``.
```

Comment:
There are a lot of `SingleTableMetadata` in the multitable files. Maybe ctrl f for them and put `MultiTableMetadata` instead.

Reply:
I think they should be ok since they are mocks of `SingleTableMetadata` or just calls from it